.github/workflows/ci.yml

```diff
@@ -549,10 +549,24 @@ jobs:
         run: ./mvnw -pl benchmarks/project jib:build -P starter
       - name: Build Worker Image
         run: ./mvnw -pl benchmarks/project jib:build -P worker
+  deploy-snyk-projects:
+    name: Deploy Snyk development projects
+    needs: [ test-summary ]
+    if: |
+      github.repository == 'camunda/zeebe' &&
+      (startsWith(github.ref, 'stable/') || github.ref == 'main')
```

Comment:
❌ `github.ref` starts with `refs/head/<branch_name>`, see also how we match for main in line 570 . An alternative might be `github.ref_name` which only contains the plain branch name

Reply:
Good catch :+1:

Reply:
I also added a check that we do it on push. Not that we do PRs for main/stable, but just in case.